src/prpy/base/robot.py

```diff
@@ -78,24 +78,32 @@ def __dir__(self):
         # __methods__ bypass __getattribute__.
         self = bind.InstanceDeduplicator.get_canonical(self)
 
-        # Add planning methods to the tab-completion list.
+        # Add planning and action methods to the tab-completion list.
         method_names = set(self.__dict__.keys())
         method_names.update(self.planner.get_planning_method_names())
+        method_names.update(self.actionlibrary.get_actions())
```

Comment:
What do you think about `self.actions` instead of `self.actionlibrary`? I'm not strongly opposed to `actionlibrary`, but it does seem a bit verbose.